packages/central-server/src/apiV2/meditrakApp/postChanges.js

```diff
@@ -36,6 +36,8 @@ const ACTIONS = {
   AddSurveyFile: 'AddSurveyFile',
 };
 
+// TODO: Tidy this up as part of RN-502
```

Comment:
`rm -rf packages/central-server/src/apiV2/meditrakApp` ? 😆

Reply:
We basically do in RN-502 :grin: